magicblock-aperture/src/requests/http/send_transaction.rs

```diff
@@ -15,8 +18,10 @@ impl HttpDispatcher {
         &self,
         request: &mut JsonRequest,
     ) -> HandlerResult {
+        let _timer = TRANSACTION_PROCESSING_TIME.start_timer();
```

Comment:
Confused about this .. where is this timer ever _stopped_? Or does that happen on `Drop`?